xdebug/README.md

```diff
@@ -0,0 +1,126 @@
+    
+    ![PhpStorm XDebug config](phpstorm-xdebug-config.png)
+
+1. Next, we need to adjust debug port to 9003 which is the default XDebug port. Do so by going to `System Preferences/Languages and Frameworks/PHP/Debug` and setting *Debug port* under XDebug to 9003.
+1. In the end we just need to start listening to debug connections: enable `Run/Start Listening for PHP Debug Connections` in the menu bar or use the shortcut button with the phone icon on the top right side of PhpStorm window.
+
+## Activating debugger
+The easiest way for activating debugger is by installing XDebug Helper browser extensions. Choose your favorite browser, install and configure extension, and you'll be able to start and stop debugging mode in your browser with just a few clicks.
+- [XDebug Helper for Firefox](https://addons.mozilla.org/en-GB/firefox/addon/xdebug-helper-for-firefox/)
+- [XDebug Helper for Chrome](https://chrome.google.com/webstore/detail/xdebug-helper/eadndfjplgieldjbigjakmdgkmoaaaoc)
+- [XDebug Helper for Safari](https://apps.apple.com/app/safari-xdebug-toggle/id1437227804?mt=12)
+
+Alternatively, if you want to activate the debugger manually, you'll need to add a trigger to your requests. Depending on what you need, add one of the following:
+- `XDEBUG_SESSION=session_name` as an additional GET (or POST) parameter if you want to initiate debugging on that single request
+- `XDEBUG_SESSION=session_name` as a cookie: debugging will now be initiated as long as that cookie is present
+
+(value of `session_name` here is not important, you can set it to whatever you like)
+
+## Test
+Finally, choose a request you want to debug, set a breakpoint in PhpStorm on the line in your code you're sure that your request will hit and fire away. If everything went well, PhpStorm should have opened debug window and now you have complete insight in what is happening in your code.
+
+![PhpStorm debug window](phpstorm-debugging.png)
+
+## Easily enable and disable XDebug
+To prevent XDebug slowing down our requests when we are not debugging, it would be wise to disable it whenever we're not using it. That's why we'll guide you to write a few shell scripts and aliases with which XDebug enabling or disabling will be a matter of entering a three-letter command in terminal.
+
+What we'll want to do is rename `xdebug.ini` file so that it can't be found and hence won't be loaded, and after that reload PHP-FPM and web server.
+
+### MacOS configuration (ZSH shell)
+Execute the following in the console:
+```
+cd /usr/local/bin
+sudo vim disable-xdebug.zsh
+```
+In the Vim editor, paste the following code:
+```
+#!/bin/zsh
+
+mv /opt/local/var/db/php74/xdebug.ini /opt/local/var/db/php74/xdebug.ini.disabled
+port reload nginx
+port reload php74-fpm
+echo "xdebug is now disabled"
+```
+Save and exit Vim.
+
+We just created our script for disabling XDebug. Let's also create a one for enabling it. Execute:
+```
+sudo vim enable-xdebug.zsh
+```
+Here, paste:
+```
+#!/bin/zsh
```

Comment:
`zsh` is the default on MacOS Big Sur, but otherwise `bash` shell is the default for writing portable scripts, as it should be available on any Unix-like OS. Also, the shebang should start with `/usr/bin/env`, which makes the actual location of the command dynamically resolved and not hardcoded. So you should have it like this:

```
#!/usr/bin/env bash
```

The actual code can stay the same since it doesn't contain anything shell-specific.

Reply:
Thanks, I fixed this in https://github.com/netgen/local-development-setup/pull/11/commits/95d0cd4ef0b5fbdc6527c1bc79eb10a8348d2fba, please check when you get a chance.